dataflow/src/main/java/org/checkerframework/dataflow/cfg/node/ObjectCreationNode.java

```diff
@@ -151,11 +150,11 @@ public boolean equals(@Nullable Object obj) {
         }
         ObjectCreationNode other = (ObjectCreationNode) obj;
         // TODO: See issue 376
```

Comment:
This PR fixes the first part of #376.
I think we said you'll make a separate PR where you try comparing only Trees for all Nodes.
How about making #376 about only this issue and open a new issue for the Tree comparison?

Reply:
Thanks!